From 036cd0052bb80d8020f3753f9c32330729552c85 Mon Sep 17 00:00:00 2001
From: Malte Marquarding <Malte.Marquarding@gmail.com>
Date: Wed, 5 Feb 2014 09:15:07 +0900
Subject: Set the SONAME of the libraries to comply with the Debian Policy

Origin: upstream, https://code.google.com/p/casacore/source/detail?r=21248
Bug: https://code.google.com/p/casacore/issues/detail?id=50
Applied-Upstream: 1.6.0
---
 CMakeLists.txt | 15 +++++++++++++--
 1 file changed, 13 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8bb137c..4c1e318 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,9 +3,14 @@
 #
 project(casacore)
 
+set(PROJECT_VERSION_MAJOR 1)
+set(PROJECT_VERSION_MINOR 5)
+set(PROJECT_VERSION_PATCH 0)
+set(PROJECT_VERSION_REVISION 1)
+set(PROJECT_VERSION
+"${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}")
+
 SET(CPACK_PACKAGE_NAME "casacore")
-SET(PROJECT_VERSION "1.4.0")
-SET(PROJECT_VERSION_REVISION 1)
 string(TOLOWER "${CPACK_PACKAGE_NAME}" CPACK_PACKAGE_NAME_LOWERCASE) 
 find_program(DPKG_PROGRAM dpkg DOC "dpkg program of Debian-based systems") 
 if(DPKG_PROGRAM) 
@@ -179,6 +184,12 @@ endif(USE_OPENMP)
 # Add the modules to be built.
 foreach (module ${_modules})
     add_subdirectory (${module})
+    set_target_properties(
+      casa_${module}
+      PROPERTIES
+          VERSION "${PROJECT_VERSION}"
+          SOVERSION "${PROJECT_VERSION_MAJOR}"
+      )
     if (APPLE)
         if (${module} STREQUAL scimath_f OR ${module} STREQUAL fits OR ${module} STREQUAL mirlib OR ${module} STREQUAL coordinates)
             set_target_properties(casa_${module} PROPERTIES LINK_FLAGS -single_module)
